app/controllers/service_provider_controller.rb

```diff
@@ -12,7 +12,7 @@ def update
   private
 
   def authorize
-    if authorization_token == AppConfig.env.dashboard_api_token
+    if authorization_token == IdentityConfig.store.dashboard_api_token
```

Comment:
While we're here, should we secure compare this?

Reply:
Yeah, good call